ProjectSettings/TagManager.asset

```diff
@@ -13,8 +13,8 @@ TagManager:
   - UI
   - 
   - 
-  - 
-  - 
+  - ButtonStopper
+  - Button
   - 
```

Comment:
Yeah this is no good, I shouldn't have added this to this commit, this was a part of the example project rather than these module modifications